src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -401,8 +606,42 @@ services.AddHttpClient(
 ```
 
 > [!NOTE]
-> The single instance returned by `HttpClientFactory` is reused by all export
-requests.
+> `IHttpClientFactory` is NOT currently supported by `OtlpLogExporter`.
+
+## Experimental features
+
+The following features are exposed experimentally in the OTLP Exporter. Features
+are exposed experimentally when either the [OpenTelemetry
+Specification](https://github.com/open-telemetry/opentelemetry-specification)
+has explicitly marked something
+[experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md)
+or when the SIG members are still working through the design for a feature and
+want to solicit feedback from the community.
+
+### Environment variables
+
+> [!NOTE]
+> In OpenTelemetry .NET environment variable keys are retrieved using
+  `IConfiguration` which means they may be set using other mechanisms such as
+  defined in appSettings.json or specified on the command-line.
+
+* All signals
+
+  * `OTEL_DOTNET_EXPERIMENTAL_OTLP_ENABLE_INMEMORY_RETRY`
```

Comment:
@vishweshbankwar I have questions regarding the name - does it mean later there will be something like "OTLP_ENABLE_ONDISK_RETRY`? What's the thinking which leads to the current name instead of `OTEL_DOTNET_EXPERIMENTAL_OTLP_ENABLE_RETRY`?